### ISSUE TITLE
[BUG] MINI - Print Fan error at 2nd or so layer but all tests passed

### ISSUE THREAD

**Issue by davcin** [prusa3d/Prusa-Firmware-Buddy#3260]
**Printer type** - [MINI]

**Printer firmware version** - 4.4.1

**Original or Custom firmware** - Original

**Optional upgrades** - Filament Runout Sensor

**USB drive or USB/Octoprint**
  USB flash drive

**Describe the bug**
  After upgrading the Firmware to 4.4.1 I started to get the Print Fan not working error. It wasn't happening before the upgrade.
  Done the self test and the fan testes and the fan works (all tests passed).
  This error stops printing since it always fails.

**How to reproduce**
  Print using the 4.4.1 firmware. I guess it doesn't happen to everyone since there would be allot of reports of that, so I assume it must be something specific to my printer.
  I can't recall from what firmware I upgraded from but I didn't do a factory reset after upgrading.

**Expected behavior**
  Print without error.
  For this, I disable the fan check, not sure if it is a good idea but had no choice.
  After disabling the fan check, the print doesn't stop and the print fan does turn on.

**G-code**
  Any G-code I tried.

**Crash dump file**
  (none)

**Video**
  (none)

**Comment by laird2205** [comment]
I have the same issue. Started after going to 4.4.1+4194. If I am looking at it correctly, the previous firmware on my usb stick is Buddy_4.3.4+2835.bbf .

**Comment by m-klecka** [comment]
Same issue on 5.1.0 alpha2, but was happening before too. Self tests show everything ok.

**Comment by evxjson** [comment]
One of our printers with 5.1.0 alpha2 has this issue. All self tests shows ok.

**Comment by Moyrn** [comment]
same issue here. i was on 4.4.0 all fine since months.
moved to 4.4.1 to instantly jump to 5.1.0 (eager to test the input shaper)
> impossible to print : print fan error

Self test is also KO on the print fan on my side as well

**Comment by Moyrn** [comment]
Strange thing : I removed the print fan and the self-test was still KO - though RPM still reportingI then noticed the yellow wire was quasi loose. touching it and it was brokeni then resoldered it, and made another self test in free air : passed OK - reporting something like 7800RPMi mounted it back fully : passed OKI renabled "check fan"I launched a print : passed OK

So possibly a pinched or loose cable in my case. Firmware is just performing tests which were not there before.

**Comment by tjjos** [comment]
Same thing is happening to me after I upgraded to 5.1.0. I really don't want to disable the fan check. Is there any other solution for this?

**Comment by Moyrn** [comment]
on my side i fixed it by resoldering a faulty yellow wire on the fan. so not a firmware issue, but a less permissive firmware - i guess

**Comment by tjjos** [comment]
That's definitely not my issue, as the fan is spinning while printing, but the error keep popping up.

**Comment by Moyrn** [comment]
ok ok.

On my side that was also what i had.
fans were running fine, even in my hand. I could see the RPM very high
during the fan test...
It was spinning then indicated "failed". i was puzzled.

So i checked, and saw that the yellow wire was "almost" loose. When
removing the sticker, the yellow wire got out.

So i suppose the fans are ok - but the RPM read value (yellow wire) raises
issues.

Sure sure. maybe not your case. there are several cases bringing to that
issue.
but i wanted to be clearer

**Comment by Makeneer** [comment]
Same issue on MK3.5 after upgrade from MK3S+

I have upgraded my Prusa i3 MK3S+ with MMU2S to Prusa i3 MK3.5 with MMU3.
Before the upgrade I had no problems with my printer and I have printed all necessary parts for the upgrade with no problem at all.

After the upgrade everything looked fine, the self-test passed with everything OK and printing was no problem until a couple of layers where printed.
After a couple of layers my printer gave an error on the print fan. I have checked the wiring and connectors, checked the working of the fan with the available menu options and did the fan test again.
Everything seems to be OK.

But my printer is still giving the same print fan error after a couple of layers.
I have changed the slicer profile so the print fan is running from the first layer and made a new test print.
The fan is running OK from the start of the print but after a couple of layers the printer is giving me the same error again.

I have checked the working of the print fan over the complete X-axis range a couple of times and it is working OK every time and every test.
e
